src/operator/ResourcesAccountType.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  ComponentColor,
+  DropdownMenuTheme,
+  MultiSelectDropdown,
+} from '@influxdata/clockface'
+import React, {FC, useContext} from 'react'
+import {OperatorContext} from './context/operator'
+import {AccountType} from 'src/types'
+
+const ResourcesAccountType: FC = () => {
+  const {accountTypes, setAccountTypes} = useContext(OperatorContext)
+  const availableTypes: AccountType[] = allAccountTypes
```

Comment:
why are we doing this if it exists as a const outside of the component?

Reply:
Good call. I'll eliminate it.